packages/dd-trace/src/telemetry/logs/log-collector.js

```diff
@@ -18,37 +18,24 @@ function hashCode (hashSource) {
 }
 
 function createHash (logEntry) {
-  if (!logEntry) return 0
-
   const prime = 31
   let result = ((!logEntry.level) ? 0 : hashCode(logEntry.level))
   result = (((prime * result) | 0) + ((!logEntry.message) ? 0 : hashCode(logEntry.message))) | 0
 
-  // NOTE: tags are not used at the moment
+  // NOTE: tags not used yet but must be of type string and comma separated value in the form 'TAG:value, TAG:value'
   // result = (((prime * result) | 0) + ((!logEntry.tags) ? 0 : hashCode(logEntry.tags))) | 0
```

Comment:
What is the goal of this lines? If we are not using tags yet, why do we need code commented and the NOTE?

Reply:
removed